x/tokenregistry/keeper/keeper.go

```diff
@@ -85,75 +85,67 @@ func (k keeper) CheckDenomPermissions(ctx sdk.Context, denom string, requiredPer
 	return true
 }
 
+func (k keeper) GetDenomPrefix(ctx sdk.Context, denom string) []byte {
+	return append(types.TokenDenomPrefix, []byte(denom)...)
+}
+
 func (k keeper) GetDenom(ctx sdk.Context, denom string) types.RegistryEntry {
```

Comment:
This func, if denom is not found, returns a RegistryEntry with Denom = denom.

Are we preserving it in this revision? I suggest adding a unit test to encapsulate this behaviour